cves/CVE-2013-0861.yml

```
@@ -90,14 +100,15 @@ fixes_vcc_instructions: |
   find interesting in there.
 fixes:
 - commit: d270c3202539e8364c46410e15f7570800e33343
-  note: ''
-- commit: ''
-  note: ''
+  note: "Fix for latest version"
+- commit: 4cd1dad91ae97fe1f0dd534c3f5566787566f137 
+  note: "Fix for version 1.0.4"
```

```suggestion
  note: Fix for version 1.0.4
```
Is there more you can elaborate on what the fix was for the latest version?

---

I noticed this on other fields you added but I'll put it here anyway, you don't need to wrap strings or other fields in quotes unless specified in the example template.